### PR TITLE
[JENKINS-39067] Move the GitHub icons to the github-branch-source plugin

### PR DESCRIPTION
**Pull request by stephenc**
See [JENKINS-39067](https://issues.jenkins-ci.org/browse/JENKINS-39067)

@reviewbybees

### CODE REVIEW

**Comment by ghost** [comment]
This pull request originates from a [CloudBees](https://www.cloudbees.com/) employee. At CloudBees, we require that all pull requests be reviewed by other CloudBees employees before we seek to have the change accepted. If you want to learn more about our process please see [this explanation](https://github.com/reviewbybees/about#about-reviewbybees).

**Comment by abayer** [comment]
Java 8 javadoc, man...

:bee:

**Comment by stephenc** [comment]
@abayer 

> error: cannot find symbol [ERROR] import org.jenkinsci.plugins.github_branch_source.GitHubLink;

Actually it's waiting on the upstream PR to be merged: https://github.com/jenkinsci/github-branch-source-plugin/pull/86

**Comment by abayer** [comment]
Aaaaah. I just assume whenever I see Javadoc failures that it's Java 8. =)

File: src/main/java/org/jenkinsci/plugins/orgfolder/github/GitHubLink.java
Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/orgfolder/github/GitHubLink.java:36]: Note that you could also use XStream aliases and completely delete classes like this, if the fields are compatible.

**Comment by svanoort** [comment]
🐝  once you silence the DocLint complaints so the tests pass (don't care if it's by disabling doclint or by fixing it -- your call there)

**Review by jglick — APPROVED** [review]
:bee:

**Comment by stephenc** [comment]
@svanoort the errors are because the build agent hasn't see the latest -SNAPSHOT of ghbs

**Comment by stephenc** [comment]
@reviewbybees done

**Pull request merged**
